scripts/dela_coverage_git_refs.py

```
@@ -0,0 +1,1081 @@
+    *,
+    runner: CommandRunner = subprocess.run,
+) -> Path:
+    _run_checked(
+        runner,
+        ["cargo", "build", "--quiet"],
+        cwd=project_root,
+        context="build dela",
+    )
+
+    if not DEFAULT_DELA_PATH.exists():
+        raise RuntimeError(f"Expected dela binary at {DEFAULT_DELA_PATH}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)